Trimble.Modus.Components/Controls/TMDataGrid/Maui.DataGrid.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why is this here? The DataGrid is not added as a separate project right?

Reply:
This is not needed, removed